api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -64,8 +64,14 @@ type VirtualMachineImageSpec struct {
 
 // VirtualMachineImageStatus defines the observed state of VirtualMachineImage
 type VirtualMachineImageStatus struct {
+	// Uuid uniquely identifies the VirtualMachineImage instance on Virtual Center
```

Comment:
The Uuid and InternalId are dependent on the source of the virtual machine image. 

For CL images, the Uuid is the content library item's ID, and internalId is the name (not sure why we have it that way, since the object's metadata.name is also the same). 

For inventory VM images, Uuid is the vCenter's VM's BiosUUID (ugh, we need to stop using that since it's not unique across vms), and InternalId is the moRef. 

We really need to do redo the status field, I've added it as an item for v1alpha2.

Reply:
Makes sense. a bunch of things to rework there. Let's not worry about fixing the comments for this now?.

Reply:
Do we just wanna add comments on top saying // Deprecated ?

Reply:
Let's do deprecated